app/assets/javascripts/discourse/app/lib/discourse-location.js

```diff
@@ -6,6 +6,17 @@ let popstateFired = false;
 const supportsHistoryState = window.history && "state" in window.history;
 const popstateCallbacks = [];
 
+function _uuid() {
```

Comment:
This 'uuid' feature is part of upstream's "history-location" routing adapter, which we eventually intend to migrate to.

https://github.com/emberjs/ember.js/blob/9e51c3629d18f53bfc057ac94987b5d27eb7cdfa/packages/%40ember/routing/history-location.ts#L12-L19